brave/brave-core
# [iOS] HF translation for 1.70.1

Resolves https://github.com/brave/brave-browser/issues/41879



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Change your device to a language other than english
2. open Brave and navigate to https://admirable-blini-a16ea7.netlify.app/
3. connect your Brave Wallet to the Dapp
4. Click Send transaction (mainnet) fromt he Dapp
5. Observe the pending transaction screen shows up and there will be a new warning message and the message should be localized.
